prowler/providers/aws/services/rds/rds_instance_event_subscription_security_groups/rds_instance_event_subscription_security_groups.py

```diff
@@ -0,0 +1,33 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_instance_event_subscription_security_groups(Check):
+    def execute(self):
+        findings = []
+        if rds_client.provider.scan_unused_services or rds_client.db_instances:
+            for db_event in rds_client.db_event_subscriptions:
+                report = Check_Report_AWS(self.metadata())
+                report.status = "FAIL"
+                report.status_extended = "RDS security group event categories of configuration change and failure are not subscribed."
+                report.resource_id = db_event.id
+                report.resource_arn = db_event.arn
```

Comment:
What are this values? If the event notification is not set we are storing it as `""` in the RDS service.

Reply:
Yeap, that is right because there is no resource that applies.

Reply:
We cannot set those values to empty strings, please review this.

Reply:
Done